### PR TITLE
HW-52575. Fixed ui tests

### PR DESCRIPTION
**Pull request by azakrevska-epam**

### CODE REVIEW

File: ui/src/androidTest/java/com/hyperwallet/android/transfermethod/ui/BankAccountTest.java [hunk 1]
```diff
@@ -141,8 +141,8 @@ public void testAddTransferMethod_displaysFeeElementsOnTmcResponse() {
                 matches(withText(R.string.add_transfer_method_fee_label)));
         onView(withId(R.id.add_transfer_method_processing_label)).check(
                 matches(withText(R.string.add_transfer_method_processing_time_label)));
-        onView(withId(R.id.add_transfer_method_fee_value)).check(matches(withText("USD 2.00")));
```
Comment (skoong) [inline ui/src/androidTest/java/com/hyperwallet/android/transfermethod/ui/BankAccountTest.java]: Leave the data the same please

File: ui/src/androidTest/java/com/hyperwallet/android/transfermethod/ui/PayPalTest.java [hunk 2]
```diff
@@ -128,8 +128,8 @@ public void testAddTransferMethod_displaysFeeElementsOnTmcResponse() {
                 matches(withText(R.string.add_transfer_method_fee_label)));
         onView(withId(R.id.add_transfer_method_processing_label)).check(
                 matches(withText(R.string.add_transfer_method_processing_time_label)));
-        onView(withId(R.id.add_transfer_method_fee_value)).check(matches(withText("USD 0.25")));
```
Comment (skoong) [inline ui/src/androidTest/java/com/hyperwallet/android/transfermethod/ui/PayPalTest.java]: Add this back as well. Currently it's not returning on qamaster but we want to try to provide mocked reponses that will be close to the ones used.

File: ui/src/androidTest/java/com/hyperwallet/android/transfermethod/ui/SelectTransferMethodTest.java [hunk 3]
```diff
@@ -146,27 +145,6 @@ public void testSelectTransferMethod_verifyCountrySelectionList() {
         onView(withId(R.id.select_transfer_method_country_value)).check(matches(withText("Canada")));
     }
 
-    @Test
-    public void testSelectTransferMethod_verifyCountrySelectionSearch() {
```
Comment (skoong) [inline ui/src/androidTest/java/com/hyperwallet/android/transfermethod/ui/SelectTransferMethodTest.java]: Why is this removed?

Comment (azakrevska-epam) [reply ui/src/androidTest/java/com/hyperwallet/android/transfermethod/ui/SelectTransferMethodTest.java]: @skoong Accidentally. Returned

File: ui/src/androidTest/java/com/hyperwallet/android/transfermethod/ui/BankAccountTest.java (hunk 1 above)
Comment (azakrevska-epam) [reply ui/src/androidTest/java/com/hyperwallet/android/transfermethod/ui/BankAccountTest.java]: @skoong returned, but I unable to to handle processing time. Where I should place it?

File: ui/src/androidTest/java/com/hyperwallet/android/transfermethod/ui/PayPalTest.java (hunk 2 above)
Comment (azakrevska-epam) [reply ui/src/androidTest/java/com/hyperwallet/android/transfermethod/ui/PayPalTest.java]: @skoong Could you assist how to provide it via mock, please?

**Comment by skoong** [comment]
Closing this PR as the fixes are in https://github.com/hyperwallet/hyperwallet-android-ui-sdk/pull/23
